README.md

```diff
@@ -36,16 +36,15 @@ gem 'delayed_job_ironmq', '2.0.0.pre1'
 Optionally: Add an initializer (`config/initializers/delayed_job.rb`):
 
 ```ruby
-Delayed::Worker.configure do |config|
-  # optional params:
-  config.available_priorities = [-1,0,1,2] # Default is [0]. Please note, adding new priorities will slow down picking the next job from queue.  Also note that these priorities must include all priorities of your Delayed Jobs.
-  config.queue_name = 'default' # Specify an alternative queue name
-  config.delay = 0  # Time to wait before message will be available on the queue
-  config.timeout = 5.minutes # The time in seconds to wait after message is taken off the queue, before it is put back on. Delete before :timeout to ensure it does not go back on the queue.
-  config.expires_in = 7.days # After this time, message will be automatically removed from the queue.
-  config.error_queue = 'error_queue' # The failed jobs will be placed into the error queue
-end
+Delayed::IronMqBackend..available_priorities = [-1,0,1,2] # Default is [0]. Please note, adding new priorities will slow down picking the next job from queue.  Also note that these priorities must include all priorities of your Delayed Jobs.
+Delayed::IronMqBackend..queue_name = 'default' # Specify a default queue name
+Delayed::IronMqBackend..delay = 0  # Time to wait before message will be available on the queue
+Delayed::IronMqBackend..timeout = 5.minutes # The time in seconds to wait after message is taken off the queue, before it is put back on. Delete before :timeout to ensure it does not go back on the queue.
+Delayed::IronMqBackend..expires_in = 7.days # After this time, message will be automatically removed from the queue.
+Delayed::IronMqBackend..error_queue = 'error_queue' # The failed jobs will be placed into the error queue
+Delayed::IronMqBackend.queues = ['first', 'second', 'third'] 
```

Comment:
could you add description and also describe how it's different from `.queue_name` ?